mesh/mesh.cpp

```diff
@@ -3395,6 +3554,272 @@ void Mesh::Make3D(int nx, int ny, int nz, Element::Type type,
    // Finalize(...) can be called after this method, if needed
 }
 
+
+void Mesh::Make2D4TrisFromQuad(int nx, int ny, double sx, double sy)
+{
+   SetEmpty();
+
+   Dim = 2;
+   spaceDim = 2;
+
+   int i, j, k;
```

Comment:
Declare loop variables in the relevant `for`, this is common to most of the loops introduced in this pr.